setup.py

```diff
@@ -82,15 +81,9 @@
         'testing': [
             'coverage',  # inspects tests coverage
             'codecov',   # sends tests coverage to codecov.io
-
             # cherrypy.lib.gctools
             'objgraph',
-
-            # pytest >= 5.4.0 doesn't work with pytest-sugar <= 0.9.2
-            # once https://github.com/Teemu/pytest-sugar/pull/188 is
-            # merged, the upper bound can be removed.
-            # Ref: https://github.com/pytest-dev/pytest/issues/6931
-            'pytest>=5.3.5,<6.1.0',
+            'pytest',
```

Comment:
Per comment, the lower bound shouldn't be removed. It should be in sync with that `pytest.ini` says.